### PR TITLE
Feature/gui

### PR DESCRIPTION
**Pull request by sebastianzabielski**
Working login and register. Dashboard only GUI skeleton.

### CODE REVIEW

File: xclient_gui/desktop/views/components/chat.py [hunk 1]
```diff
@@ -1,48 +1,38 @@
...
-        # hbox.direction = QVBoxLayout.BottomToTop
+        layout = QVBoxLayout()
 
-        hbox.addWidget(tree)
-        hbox.addWidget(tree2)
+        messages = MessagesList()
 
-        self.setLayout(hbox)
-        self.show()
+        messages.addItem("Witaj w pokoju Test1.\nLista aktywnych użytkowników: Jan Kowalski, Pan Zbysiu, Zenek")
```
Comment (RaVkloc) [inline xclient_gui/desktop/views/components/chat.py:31]: Text in English, please

File: xclient_gui/desktop/views/register.py [hunk 2]
```diff
@@ -62,27 +53,29 @@ def create_password_input(self, layout):
...
 
-    def create_login_button(self, layout):
-        button_login = QPushButton('Login')
-        button_login.clicked.connect(self.login)
-        button_login.setStyleSheet("padding-top: 5px; padding-bottom: 5px;")
-        layout.addWidget(button_login, 2, 0, 1, 2)
-        layout.setRowMinimumHeight(2, 75)
+    def create_confirm_password_input(self, layout):
+        label_confirm_password, self.lineEdit_confirm_password = FormInput('Confirm Password:',
+                                                                           "Confirm your password").get_input()
```
Comment (RaVkloc) [inline xclient_gui/desktop/views/register.py:59]: Add ":" to get "Confirm your password:"

File: xclient_gui/desktop/views/register.py [hunk 3]
```diff
@@ -62,27 +53,29 @@ def create_password_input(self, layout):
...
         layout.addWidget(self.lineEdit_password, 1, 1)
 
-    def create_login_button(self, layout):
-        button_login = QPushButton('Login')
-        button_login.clicked.connect(self.login)
-        button_login.setStyleSheet("padding-top: 5px; padding-bottom: 5px;")
-        layout.addWidget(button_login, 2, 0, 1, 2)
-        layout.setRowMinimumHeight(2, 75)
+    def create_confirm_password_input(self, layout):
+        label_confirm_password, self.lineEdit_confirm_password = FormInput('Confirm Password:',
```
Comment (RaVkloc) [inline xclient_gui/desktop/views/register.py]: Password should be lowercase

File: xclient_gui/desktop/views/components/chat.py [hunk 4]
```diff
@@ -1,48 +1,38 @@
...
-        tree2 = Tree()
-        tree2.setFrameShape(QFrame.StyledPanel)
-        tree2.setStyleSheet('background-color:red')
+class NewMessageInput(QHBoxLayout):
+    def __init__(self):
+        super().__init__()
 
-        # chat = Chat()
-        # chat.setStyleSheet('background-color:green')
+        message_input = QLineEdit()
```
Comment (RaVkloc) [inline xclient_gui/desktop/views/components/chat.py:17]: Consider replacing QLineEdit to multiline QPlainTextEdit

**Review by RaVkloc — APPROVED** [review]

File: xclient_gui/desktop/views/register.py (hunk 2 above)
Comment (sebastianzabielski) [reply xclient_gui/desktop/views/register.py:59]: Confirm your password is a placeholder. There is no need to add ":" in placeholder.

File: xclient_gui/desktop/views/components/chat.py (hunk 1 above)
Comment (sebastianzabielski) [reply xclient_gui/desktop/views/components/chat.py:31]: This text is only temporary, to test if everything is displaying right and will be removed in the next pull request.

File: xclient_gui/desktop/views/register.py (hunk 3 above)
Comment (sebastianzabielski) [reply xclient_gui/desktop/views/register.py]: Fixed.

File: xclient_gui/desktop/views/components/chat.py (hunk 4 above)
Comment (sebastianzabielski) [reply xclient_gui/desktop/views/components/chat.py:17]: I will try it. A possible change will appear in the future pull request.

**Pull request merged**